rllib/algorithms/algorithm_config.py

```diff
@@ -2634,22 +2650,53 @@ def validate_train_batch_size_vs_rollout_fragment_length(self) -> None:
             batch_size = min_batch_size
             while batch_size < self.train_batch_size:
                 batch_size += min_batch_size
+
+            # suggested_rollout_fragement should
+            # train_batch_size // (num_envs_per_worker * num_rollout_workers) (+ 1)
+            # the plus 1 is for making sure that rollout_fragment_length *
```

Comment:
Self review: make it clear that this plus 1 depends on whether the train_batch_size is already a multiple integer or not.